tests/unit/test_main.py

```diff
@@ -0,0 +1,366 @@
+        (mock_get_data, mock_get_interfaces, mock_get_interfaces_dict) = args
+
+        topology = get_topology_mock()
+        interfaces = self.get_topology_interfaces(topology)
+        interfaces_dict = {inter.id: inter for inter in interfaces}
+        interfaces_ids = [inter.id for inter in interfaces]
+
+        mock_get_data.return_value = interfaces_ids
+        mock_get_interfaces.return_value = interfaces
+        mock_get_interfaces_dict.return_value = interfaces_dict
+
+        api = get_test_client(self.napp.controller, self.napp)
+
+        url = "http://127.0.0.1:8181/api/kytos/of_lldp/v1/interfaces/disable"
+        disable_response = api.open(url, method='POST')
+
+        url = "http://127.0.0.1:8181/api/kytos/of_lldp/v1/interfaces/enable"
+        enable_response = api.open(url, method='POST')
+
+        self.assertEqual(disable_response.status_code, 200)
+        self.assertEqual(enable_response.status_code, 200)
+
+    @patch('napps.kytos.of_lldp.main.Main._get_interfaces')
+    @patch('napps.kytos.of_lldp.main.Main._get_data')
+    def test_enable_disable_lldp_404(self, *args):
+        """Test 404 response for enable_lldp and disable_lldp methods."""
+        (mock_get_data, mock_get_interfaces) = args
+
+        mock_get_data.return_value = []
+        mock_get_interfaces.return_value = None
+
+        api = get_test_client(self.napp.controller, self.napp)
+
+        url = "http://127.0.0.1:8181/api/kytos/of_lldp/v1/interfaces/disable"
+        disable_response = api.open(url, method='POST')
+
+        url = "http://127.0.0.1:8181/api/kytos/of_lldp/v1/interfaces/enable"
+        enable_response = api.open(url, method='POST')
+
+        self.assertEqual(disable_response.status_code, 404)
+        self.assertEqual(enable_response.status_code, 404)
+
+    @patch('napps.kytos.of_lldp.main.Main._get_interfaces_dict')
+    @patch('napps.kytos.of_lldp.main.Main._get_interfaces')
+    @patch('napps.kytos.of_lldp.main.Main._get_data')
+    def test_enable_disable_lldp_400(self, *args):
+        """Test 400 response for enable_lldp and disable_lldp methods."""
+        (mock_get_data, mock_get_interfaces, mock_get_interfaces_dict) = args
+
+        topology = get_topology_mock()
+        interfaces = self.get_topology_interfaces(topology)
+        interfaces_dict = {inter.id: inter for inter in interfaces}
+        interfaces_ids = [inter.id for inter in interfaces]
+        interfaces_ids.append("00:00:00:00:00:00:00:01:3")
+
+        mock_get_data.return_value = interfaces_ids
+        mock_get_interfaces.return_value = interfaces
+        mock_get_interfaces_dict.return_value = interfaces_dict
+
+        api = get_test_client(self.napp.controller, self.napp)
+
+        url = "http://127.0.0.1:8181/api/kytos/of_lldp/v1/interfaces/disable"
+        disable_response = api.open(url, method='POST')
+
+        url = "http://127.0.0.1:8181/api/kytos/of_lldp/v1/interfaces/enable"
```

Comment:
Suggestion: use a `self.server_name_url` variable like in kytos/mef_eline/tests/test_main.py